test/one/profiler/test/Filter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.profiler.test;
+
+import java.util.regex.Pattern;
+
+public abstract class Filter {
```

Comment:
It's now a single-method utility class; I'd just move the method to RunnerDeclaration.